src/analysis/seminorm.lean

```diff
@@ -348,6 +394,23 @@ begin
   ...    < r   : by rwa mem_ball_zero at hy,
 end
 
+lemma finset_sup_ball_inter (p : ι → seminorm 𝕜 E) (ι' : finset ι) (r : ℝ) (hr : 0 < r):
```

Comment:
This is a very confusing variable name. `ι'` is an indexing type. `s` is a finset.
```suggestion
lemma finset_sup_ball_inter (p : ι → seminorm 𝕜 E) (s : finset ι) (r : ℝ) (hr : 0 < r) :
```